### PR TITLE
Weather App

### PR DESCRIPTION
**Pull request by evaveskova**
In this project I: 

- Set up a basic page using HTML and CSS.
- Set up a form to pull information from the openweathermap API based on the inputted city.
- Created a function in JavaScript to display the inputted city with it's weather type and temperature.
- Set the background image to change depending on the weather type.

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      setBackground(weather[0].main);
+
+      const weatherTemperature = document.getElementById('weather-temp');
+      weatherTemperature.textContent = '';
+      const temperatureResult = document.createElement('H3');
+      temperatureResult.textContent = `${main.temp}°C`;
+      weatherTemperature.appendChild(temperatureResult);
+
+      document.getElementById('city-input').value = '';
+    });
```
Comment (Oghenebrume50) [inline src/api.js]: Please add error-catching for this see this

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "homepage": "https://github.com/evaveskova/weather_app#readme",
+  "devDependencies": {
+    "css-loader": "^3.4.2",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.20.0",
+    "file-loader": "^5.0.2",
+    "style-loader": "^1.1.3"
+  },
```
Comment (Oghenebrume50) [inline package.json]: Add all necessary webpack packages to devdependencies see this
see above script object for hint on the needed packages

File: package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "weather_app",
+  "version": "1.0.0",
+  "description": "The aim of this project was to create a weather app where users can input a city and view the weather type and temperature of that city. The original project requirements can be found on [The Odin Project](https://www.theodinproject.com/courses/javascript/lessons/weather-app).",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (Oghenebrume50) [inline package.json]: setup script to work properly with webpack see

`"scripts": {`
    `"dev": "webpack --mode development",`
    `"build": "webpack --mode production",`
    `"start": "webpack-dev-server --mode development --open" // optional`
  `},`

**Comment by evaveskova** [comment]
All requested changes have been made.

**Pull request merged**